ecl/hql/hqlexpr.hpp

```diff
@@ -727,6 +727,12 @@ enum node_operator : unsigned short {
         no_fromjson,
         no_tojson,
         no_matched_injoin,
+        no_httppost,
```

Comment:
I wonder whether it might be simpler/cleaner to reuse the soapcall nodes.  The only complication would be regenerating ecl code.

Reply:
yes, I started that way, thought about regeneration and maybe intuitive reading of graphs?